app/services/RestaurantService.java

```diff
@@ -0,0 +1,63 @@
+package services;
+
+import models.Restaurant;
+import org.hibernate.Criteria;
+import org.hibernate.Session;
+import org.hibernate.jpa.HibernateEntityManager;
+import play.db.jpa.JPA;
+import repository.RestaurantRepository;
+
+import javax.inject.Inject;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by lapa on 5/17/17.
+ */
+public class RestaurantService implements BaseService{
+
+    private RestaurantRepository restaurantRepository;
+
+    @Inject
+    public void setRestaurantRepository(RestaurantRepository restaurantRepository) {
+        this.restaurantRepository = restaurantRepository;
+    }
+
+    @Override
+    public Criteria getCriteria() {
+        Session session = ((HibernateEntityManager) JPA.em()).getSession();
+
+        return session.createCriteria(Restaurant.class);
+    }
+
+    public List<Restaurant> getRestaurants(){
+        List<Restaurant> restaurants = getCriteria().list();
```

Comment:
Every method that interacts with database should be in repository class, so this line should call e.g. `repository.findAll()` method.
Also `findAll` method can be implemented in your base repository implementation.